.circleci/config.yml

```diff
@@ -206,7 +206,6 @@ commands:
           command: |
             cf push clamav-rest -f tdrs-backend/manifest.clamav.yml \
               --var cf-space=<<parameters.cf-space>> \
-              --strategy rolling
```

Comment:
This was removed due to a memory quota limit in the dev space. Just pushing the manifest will use less memory than the rolling strategy but will introduce between 30s and 2m of downtime while the app gets stood up.